app/helpers/application_helper.rb

```diff
@@ -1,4 +1,16 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  def error_message(field:, resource:)
```

Comment:
I think `error_message` is a too generic name for what this does.

Maybe we could rename it to `field_error`?